pkg/backend/httpstate/state.go

```diff
@@ -183,7 +183,7 @@ func (b *cloudBackend) newUpdate(ctx context.Context, stackRef backend.StackRefe
 				ctx, update, currentToken, duration)
 			if err != nil {
 				// Translate 403 status codes to expired token errors to stop the token refresh loop.
-				var apierr apitype.ErrorResponse
+				var apierr *apitype.ErrorResponse
```

Comment:
I've also made this same mistake with `errors.As` more than once I think 😭 So easy to get wrong.